src/elements/header/header-link/header-link.visual.spec.ts

```diff
@@ -0,0 +1,33 @@
+import { html } from 'lit';
+
+import { describeViewports, visualDiffStandardStates } from '../../core/testing/private.js';
+
+import './header-link.js';
+
+describe(`sbb-header-link`, () => {
+  describeViewports({ viewports: ['zero', 'medium'] }, () => {
+    for (const state of visualDiffStandardStates) {
+      it(
+        state.name,
+        state.with(async (setup) => {
+          await setup.withFixture(html`
+            <sbb-header-link icon-name="hamburger-menu-small">Menu</sbb-header-link>
+          `);
+        }),
+      );
+
+      it(
+        `multiple=true ${state.name}`,
```

Comment:
Does the multiple add any value?
I don't see any CSS rule in the header that affects multiple elements